google/cloud/pubsub/README.md

```diff
@@ -19,12 +19,12 @@ Please note that the Google Cloud C++ client libraries do **not** follow
 
 ## Documentation
 
-* Official documentation about the [Cloud Pub/Sub][cloud-spanner-docs] service
+* Official documentation about the [Cloud Pub/Sub][cloud-pubsub-docs] service
```

Comment:
This is a generated file, you need to fix the generator too.

Reply:
ah. thanks